devices/tuya.js

```diff
@@ -141,9 +141,9 @@ module.exports = [
         model: 'TS0601_smart_air_house_keeper',
         vendor: 'TuYa',
         description: 'Smart air house keeper',
-        fromZigbee: [fz.tuya_air_quality],
+        fromZigbee: [fz.tuya_air_housekeeper],
         toZigbee: [],
-        exposes: [e.temperature(), e.humidity(), e.co2(), e.voc(), e.formaldehyd(), e.pm25()],
+        exposes: [e.temperature(), e.humidity(), e.co2(), e.voc(), e.formaldehyd().withUnit('ppm'), e.pm25()],
```

Comment:
Is the unit of formaldehyd always ppm? If yes, please update `e.formaldehyd()` in `exposes` directly (since it has no unit there yet)

Reply:
I'm not sure, if it's mainly in ppm. That's the reason, why I didn't changed that in exposes directly. I have only one device-type to get the units. It will be helpfull, if others can give a feedback about the units used by tuya. After that, we can update in exposes directly.